packages/@aws-cdk/aws-ecs/rosetta/default.ts-fixture

```diff
@@ -1,6 +1,6 @@
 // Fixture with packages imported, but nothing else
 import { Construct } from 'constructs';
-import { SecretValue, Stack } from '@aws-cdk/core';
+import * as cdkCore from '@aws-cdk/core';
```

Comment:
This won't translate to V2 properly, I don't think, so this line shouldn't be changed.

Reply:
I see - I can revert that.

The only thing I need here is `Size` 😅 I was trying to make it clearer that it comes from the core package.